.builds/ubuntu-focal.yml

```diff
@@ -3,11 +3,14 @@ packages:
   - automake
   - autopoint
   - xsltproc
+  - libbsd0
```

Comment:
Please do not add libbsd0, as that is an implementation detail of the ABI which is not relevant during builds. The libbsd-dev build-dependency is enough and will make sure to pull in whatever specific shared library is required.

Reply:
Removed `libbsd0` as suggested